mantle/cmd/kola/testiso.go

```diff
@@ -53,46 +53,83 @@ var (
 
 	instInsecure bool
 
-	nopxe bool
-	noiso bool
-
 	scenarios []string
+	pltfrm    string
 
 	pxeAppendRootfs bool
 	pxeKernelArgs   []string
 
 	console bool
 
-	addNmKeyfile bool
+	addNmKeyfile	 bool
+	enable4k	 bool
+	enableMultipath	 bool
+	enableUefi	 bool
+	enableUefiSecure bool
```

Comment:
Is there a reason to have these global vars instead of having them be scoped to the iteration loop only?

Reply:
It also needs to be visible  to all the tests functions

Reply:
I'm not thrilled about having them be globals either. What if we ever started to try to run these in parallel? Probably would be better to create a new data structure to carry this information and pass it around as arguments to the functions.

This type of change would be a good candidate for a followup and definitely doesn't need to block this code.